.gitignore

```diff
@@ -13,4 +13,3 @@
 /tmp
 /coverage
 .env
-/public/uploads
```

Comment:
Don't delete this line. We don't want to commit files uploaded to the filesystem.